src/hotspot/share/gc/g1/g1CollectedHeap.cpp

```diff
@@ -1506,6 +1506,7 @@ void G1CollectedHeap::ref_processing_init() {
   //     * Reference discovery will not need a barrier.
 
   // Concurrent Mark ref processor
```

Comment:
The comment now seems misplaced

Reply:
Moved back the comment to the original position.